lib/logstash/filters/dns.rb

```diff
@@ -42,8 +42,8 @@ class LogStash::Filters::DNS < LogStash::Filters::Base
   # specified under `reverse` and `resolve`.
   config :action, :validate => [ "append", "replace" ], :default => "append"
 
-  # Use custom nameserver.
-  config :nameserver, :validate => :string
+  # Use custom nameserver(s).
+  config :nameservers, :validate => :array
```

Comment:
I don't want to break backwards compat.

In similar cases, we've simply set `:validate => :array` on what was previously `:string` and everything worked reasonably well.

Keeping the name as `nameserver` and making it `:array` should be all you need.

This will provide your desired behavior as well as keeping backwards compat.